PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -23,6 +23,13 @@
         /// Default: true
         @objc public var captureNetworkTelemetry: Bool = true
 
+        /// By default Session replay will capture all the views on the screen as a wireframe,
+        /// By enabling this option, PostHog will capture the screenshot of the screen.
+        /// The screenshot may contain sensitive information, use with caution.
+        /// Experimental support
+        /// Default: false
+        @objc public var screenshot: Bool = false
```

Comment:
nit: maybe a nicer name here, `screenhshotMode` ???, despite being experimental we're a little stuck with it once it's in the wild 🤷